spec/helpers/filters_helper_spec.rb

```diff
@@ -1,4 +1,5 @@
 require "rails_helper"
+require "timecop"
```

Comment:
I don't think you need this here, but you might need to `include CollectionTimeHelper` in `FiltersHelper`

Reply:
You're right, thanks